go/adbc/driver/flightsql/flightsql_adbc_test.go

```diff
@@ -922,3 +929,35 @@ func (suite *TLSTests) TestInvalidOptions() {
 	_, _, err = stmt.ExecuteQuery(suite.ctx)
 	suite.Contains(err.Error(), "Unavailable")
 }
+
+func TestDuckDBFlightSQL(t *testing.T) {
```

Comment:
Do we need to add the DuckDB server to CI then? (There isn't an official one right?)

Reply:
LOL this is an accident. I did this when i was testing something for Phil Moore's Apache Superset ADBC demo and forgot to remove this. oops haha. I'll remove it.